Cont. - Add Proto Message Extraction http filter (#35689)
envoyproxy/envoy: docs/root/configuration/http/http_filters/proto_message_scrubbing_filter.rst

.. _config_http_filters_proto_message_scrubbing:

Proto Message Scrubbing
=======================

* gRPC :ref:`architecture overview <arch_overview_grpc>`
* This filter should be configured with the type URL ``type.googleapis.com/envoy.extensions.filters.http.proto_message_scrubbing.v3.ProtoMessageScrubbingConfig``.
* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.proto_message_scrubbing.v3.ProtoMessageScrubbingConfig>`.

> ProtoMessageScrubbing filter supports scrubbing scrubbed gRPC
> requests/responses (proto messages) as google.protobuf.Struct and storing
> results in the dynamic metadata `envoy.filters.http.proto_message_scrubbing`
> for later access.

Reviewer: ```suggestion
ProtoMessageScrubbing filter supports scrubbing gRPC
requests/responses (proto messages) into google.protobuf.Struct and storing
results in the dynamic metadata `envoy.filters.http.proto_message_scrubbing`
for later access.
```

Make this same change in the API docs also.

Author: Sure, done.

Thank you!
